packages/calling-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -92,6 +100,31 @@ export const createDefaultCallingHandlers: CreateDefaultCallingHandlers = memoiz
     ): Promise<void> => {
       const participant = _toCommunicationIdentifier(userId);
       await call?.removeParticipant(participant);
+    },
+    /* @conditional-compile-remove(one-to-n-calling) */
+    onAcceptCall: async (incomingCallId: string, useVideo?: boolean): Promise<void> => {
+      const cameras = await deviceManager?.getCameras();
+      let localVideoStream: LocalVideoStream | undefined;
+      if (cameras && useVideo) {
+        localVideoStream = new LocalVideoStream(cameras[0]);
+      }
```

Comment:
This part seems similar to start call - is it possible to extra this part into a common function?